integrationtest/pg/pg_test.go

```diff
@@ -55,167 +54,175 @@ func TestCreateBucket(t *testing.T) {
 }
 
 func TestArchiveTracker(t *testing.T) {
-	_ = spinup(t)
-	ctx, conf, client, shutdown := setup(t)
+	util.RunFlaky(t, func(t *util.FlakyT) {
```

Comment:
Indents from copy+paste? There's a bunch in this file.

Reply:
Oh, diff looks terrible. Nothing about the test changed. Yep, the indentation bumped since test logic is wrapped in `util.RunFlaky`.

Reply:
Ahhhh, I see